src/useRouteParam/README.md

```diff
@@ -0,0 +1,20 @@
+# useRouteParam
+The `useRouteParam` gives you a readonly reactive value for a query param. If you need the param to be writable, consider [useRouteQueryParam](https://github.com/PrefectHQ/vue-compositions/tree/main/src/useRouteQueryParam).
```

Comment:
It doesn't give you access to query params. It gives you the value of a route param. They're different. 

route.params[param]

Vs

route.query[param]

Reply:
already fixed